dataprofiler/data_readers/data_utils.py

```diff
@@ -553,12 +555,21 @@ def load_as_str_from_file(file_path, file_encoding, max_lines=10,
             
             # Return either the last index of sample_lines OR
             # the index of the newline char that matches remaining_lines
+            search_query_value = '\n'
+            if (isinstance(sample_lines, bytes)):
+                search_query_value = b'\r\n'
+            
             loc, occurance = find_nth_loc(sample_lines,
-                                          search_query='\n',
-                                          n=remaining_lines)
+                                        search_query=search_query_value,
+                                        n=remaining_lines)
 
             # Add sample_lines to data_as_str no more than max_lines
-            data_as_str += sample_lines[:loc]
+            if (is_stream_buffer(file_path) and isinstance(sample_lines[:loc],\
```

Comment:
instead of breaking it up mid function, i think it should break at the `and` to make it more readable